source/Engine.cpp

```diff
@@ -142,17 +151,26 @@ namespace {
 			Angle gimbal = Angle(gimbalDirection * point.gimbal.Degrees());
 			Angle flareAngle = ship.Facing() + point.facing + gimbal;
 			Point pos = ship.Facing().Rotate(point) * ship.Zoom() + ship.Position();
-			// If multiple engines with the same flare are installed, draw up to
-			// three copies of the flare sprite.
+			auto DrawFlares = [&draw, &pos, &ship, &flareAngle, &point](auto it, auto scale)
```

Comment:
Don't use `auto` for `Point`.
And I think it would be quite useful to explicitly state the type of `it` here because a reader will have to either guess or go further down and look at wherever this method is being called to figure out the type. I don't think it would be over 120 characters.
Also, shouldn't you be taking a reference for that, not a copy?